health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppLocalisationTable.js

```
@@ -21,40 +22,42 @@ const Tabs = ({ availableLocales, onTabChange, setActiveLocale, activeLocale })
     </div>
   );
 };
-export const AppLocalisationTable = ({ data }) => {
+export const AppLocalisationTable = ({ data, currentScreen, state }) => {
   const { locState, addMissingKey, updateLocalization } = useAppLocalisationContext();
+  const currentLocState = useMemo(() => {
+    return locState?.filter((i) => i.code && i.code.includes(currentScreen));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)